azure-pipelines.yml

```diff
@@ -15,25 +6,31 @@ pr:
 - master
 
 # Three phases for each of the three OSes we want to run on
-phases:
-- template: /eng/common/templates/phases/base.yml
+jobs:
+- template: /eng/common/templates/job/job.yml
   parameters:
     name: Windows_NT
+    enableMicrobuild: true
```

Comment:
This is for signing, I don't think you're signing, you can delete this

Reply:
Do I not want to sign darc?

Reply:
you probably do, but your current iteration doesn't appear to be telling it to actually do that.  ie, there's no signtype specified

Reply:
Is it... bad to do it premptively? I'll definitely be doing that super soon, and would like to avoid having to make a bunch of changes if I can avoid it. :-)

Reply:
Nope, it's fine, you can leave it

Reply:
Oh phew. I don't really understand what I'm doing... I'm mostly just copy pasting from samples. :-) The new jobs thing seems to be quite a bit easier to use than the other thing.